pkg/util/helper/taint_test.go

```diff
@@ -357,3 +385,132 @@ func TestGetNoExecuteTaints(t *testing.T) {
 		})
 	}
 }
+
+func TestGetMinTolerationTime(t *testing.T) {
+	noExecuteTaints := []corev1.Taint{
+		{
+			Key:       "key1",
+			Value:     "value1",
+			Effect:    corev1.TaintEffectNoExecute,
+			TimeAdded: &metav1.Time{Time: time.Now()},
+		},
+		{
+			Key:       "key2",
+			Value:     "value2",
+			Effect:    corev1.TaintEffectNoExecute,
+			TimeAdded: &metav1.Time{Time: time.Now()},
+		},
+	}
+
+	usedTolerations := []corev1.Toleration{
+		{
+			Key:               "key1",
+			Operator:          corev1.TolerationOpExists,
+			Effect:            corev1.TaintEffectNoExecute,
+			TolerationSeconds: &[]int64{60}[0],
+		},
+		{
+			Key:               "key2",
+			Operator:          corev1.TolerationOpExists,
+			Effect:            corev1.TaintEffectNoExecute,
+			TolerationSeconds: &[]int64{120}[0],
+		},
+	}
+
+	t.Run("no noExecuteTaints", func(t *testing.T) {
+		result := GetMinTolerationTime([]corev1.Taint{}, usedTolerations)
+		if result != -1 {
+			t.Errorf("Expected -1 but got %v", result)
+		}
+	})
+
+	t.Run("no usedTolerations", func(t *testing.T) {
+		result := GetMinTolerationTime(noExecuteTaints, []corev1.Toleration{})
+		if result != 0 {
+			t.Errorf("Expected 0 but got %v", result)
+		}
+	})
+
+	t.Run("with noExecuteTaints and usedTolerations", func(t *testing.T) {
+		expectedSec := 60
+		result := GetMinTolerationTime(noExecuteTaints, usedTolerations)
+		if result > time.Duration(expectedSec+1)*time.Second || result < time.Duration(expectedSec-1)*time.Second {
+			t.Errorf("Expected about %vs but got %v", expectedSec, result)
+		}
+	})
+}
+
+func TestGetMatchingTolerations(t *testing.T) {
+	taints := []corev1.Taint{
+		{
+			Key:    "key1",
+			Value:  "value1",
+			Effect: corev1.TaintEffectNoSchedule,
+		},
+		{
+			Key:    "key2",
+			Value:  "value2",
+			Effect: corev1.TaintEffectNoSchedule,
+		},
+	}
+
+	tolerations := []corev1.Toleration{
+		{
+			Key:    "key1",
+			Value:  "value1",
+			Effect: corev1.TaintEffectNoSchedule,
+		},
+		{
+			Key:    "key2",
+			Value:  "value2",
+			Effect: corev1.TaintEffectNoSchedule,
+		},
+	}
+	expected := true
+	expectedTolerations := tolerations
+	actual, actualTolerations := GetMatchingTolerations(taints, tolerations)
+	if actual != expected {
```

Comment:
Here can simply `expected` to true.

In addition, can more test items be added to cover the test path?